src/pages/ResetPassword/ResetPassword.css

```diff
@@ -1,7 +1,7 @@
 body {
   background: #e8edf1;
 }
-#loginForm {
+#resetpasswordForm {
```

Comment:
this ID cuts across all the forms on the app. Hope you checked if it doesn't affect other forms?